### DISCUSSION TITLE
loading sprites/tiles from 6502 memory

### DISCUSSION THREAD

**Discussion by bocianu** [paulscottrobson/neo6502-firmware#327] [Implemented Ideas]
For now, we are only able to load sprite data from an external file.
But what if I would like to create them dynamically from within the running code?

Or if I would like to create a native sprite/tile editor, then loading sprite data from memory (to get a live preview) would also be a great solution.

For now, I can do a workaround: prepare a .gfx file structure in 6502 memory, save it to a temporary file, and then load it into $ffff.
But you need that temporary file, and the file operation is probably slower than the mem->mem operation.

**Comment by erlendnagel** [comment]
I like the idea. I was trying out Spoon and it is a really cool sprite editor.
